src/main/java/com/google/devtools/build/lib/analysis/Dependency.java

```diff
@@ -144,14 +144,6 @@ public static Dependency withConfiguredAspects(
         ImmutableList.of());
   }
```

Comment:
I don't know why I can't add a comment at that line, but let's update the javadocs at line 32 on.

Reply:
Good point, done.